projects/packages/jetpack-mu-wpcom/src/features/wpcom-post-list/js/wpcom-post-list-tracks.ts

```diff
@@ -0,0 +1,69 @@
+import { wpcomTrackEvent } from '../../../common/tracks';
+
+declare const wpcomPostListData: {
+	postType: string;
+};
+
+/**
+ * Tracks clicks on the post list table.
+ */
+function wpcomPostListTracks() {
+	document.getElementById( 'the-list' )?.addEventListener( 'click', function ( event ) {
+		const target = event.target as HTMLElement;
+		if ( target.tagName === 'A' || target.className === 'button-link editinline' ) {
```

Comment:
Same concert about the classname:

```suggestion
		if ( target.matches( 'a,.button-link.editinline' ) ) {
```